base/io.jl

```diff
@@ -172,13 +172,13 @@ Consistent serialization:
 ```jldoctest
 julia> open("/tmp/test.bin","w") do f
            # Make sure we write 64bit integer in Network (big-endian) byte order
```

Comment:
need to change this comment too

Reply:
I really do hope I got it right this time... Sorry again!